src/SFA.DAS.EmployerIncentives/Infrastructure/Api/RestApiClient.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.WebUtilities;
+using SFA.DAS.EmployerIncentives.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Infrastructure.Api
+{
+    public class RestApiClient : IRestApiClient
```

Comment:
Is there any reason the differences in here can't be added to the ApiClient in shared? from what i can tell the only difference is the Cancellation token which is just being set to default - not changed - and is the default for if you don't pass it.

Reply:
No it's quite different from ApiClient. We are adding the authentication and logging as HttpMessageHandlers and this seems to make the intent of the each layer clearer.